pulp_service/pulp_service/app/settings.py

```diff
@@ -8,4 +8,5 @@
 SUBSCRIPTION_API_URL = "https://subscription.stage.api.redhat.com/svcrest/subscription/v5/featureStatus"
 SUBSCRIPTION_API_CERT = ""
 AUTHENTICATION_HEADER_DEBUG = False
-INSTALLED_APPS = "@merge django.contrib.admin.apps.SimpleAdminConfig"
\ No newline at end of file
+INSTALLED_APPS = "@merge django.contrib.admin.apps.SimpleAdminConfig"
+TEST_TASK_INGESTION = False
```

Comment:
This can be taken out (I think) because with it not being part of any domain, only an admin could authorize to run it with the middleware that's in place.